runelite-client/src/main/java/net/runelite/client/plugins/opponentinfo/OpponentInfoConfig.java

```diff
@@ -52,4 +52,15 @@ default boolean showPercent()
 	{
 		return false;
 	}
+
+	@ConfigItem(
+		keyName = "showOpponentsOpponent",
+		name = "Show Opponents Opponent",
```

Comment:
Maybe `Show enemy target` would be better? Also, title casing everything in here do not fits with naming of other config options